src/ipahealthcheck/core/files.py

```diff
@@ -12,6 +12,19 @@
 
 logger = logging.getLogger()
 
+EXPECTED_UMASK = 0o022
+
+
+def current_umask():
+    """
+    Retrieves current umask by setting it temporarily
+
+    :returns: int umask
+    """
+    umask = os.umask(0)
+    os.umask(umask)
```

Comment:
I'd  prefer if you'd set it to 022 instead of 0 because if something were to go wrong the permissions would be wide open. It is probably perfectly safe and healthcheck is single-threaded it just gives me the heebie jeebies.